<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>support</class>
 <widget class="QWidget" name="support">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Support</string>
  </property>
  <widget class="QLabel" name="title_text">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>17</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tell us what is on your mind</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="support_message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>53</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>306</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>306</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="character_limit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>45</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>265</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
